### PR TITLE
Sarutabaruta Region NM Audit

### PR DESCRIPTION
**Pull request by CriticalXI**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

- Implemented missing NM Brazen Bones
- Duke Decapod no longer stands back and casts water frequently.
- Corrected HP, level and immunities for several NMs.
- Maltha now has a much higher attack speed.
- Edited all drop lists to utilize the TH variables and correct drop rates according to FFXIDB.
- Edited all lottery NMs with a random respawn range to instead have a minimum respawn.

### CODE REVIEW

File: scripts/zones/Giddeus/mobs/Yagudo_Persecutor.lua [hunk 1]
```diff
@@ -11,7 +11,7 @@ entity.onMobDeath = function(mob, player, isKiller)
 end
 
 entity.onMobDespawn = function(mob)
-    xi.mob.phOnDespawn(mob, ID.mob.JUU_DUZU_THE_WHIRLWIND_PH, 5, math.random(3600, 7200)) -- 1 to 2 hours
+    xi.mob.phOnDespawn(mob, ID.mob.JUU_DUZU_THE_WHIRLWIND_PH, 5, 3600) -- 1 to 2 hours
```
Comment (TeoTwawki) [inline scripts/zones/Giddeus/mobs/Yagudo_Persecutor.lua:14]: missed updating the comment :)

also anecdotal but I'm pretty sure he and mjoo have the same spawn rate as I usually get them back to back when I've been killing their ph's. I run the circuit and boomerang his ph from the ledge on my way to vu puqu the beguiler and usually get all 3 if nobody else messes up my rotation

File: scripts/zones/Giddeus/mobs/Yagudo_Piper.lua [hunk 2]
```diff
@@ -12,7 +12,7 @@ entity.onMobDeath = function(mob, player, isKiller)
 end
 
 entity.onMobDespawn = function(mob)
-    xi.mob.phOnDespawn(mob, ID.mob.VUU_PUQU_THE_BEGUILER_PH, 5, 900) -- 15 minutes
+    xi.mob.phOnDespawn(mob, ID.mob.VUU_PUQU_THE_BEGUILER_PH, 15, 900) -- 15 minutes
```
Comment (TeoTwawki) [inline scripts/zones/Giddeus/mobs/Yagudo_Piper.lua:15]: 15min??

see above, I don't think I've ever seen him that fast myself, but I guess its possible. if it is I'd guess he's lower rate tho

File: scripts/zones/Giddeus/mobs/Yagudo_Persecutor.lua (hunk 1 above)
Comment (TeoTwawki) [reply scripts/zones/Giddeus/mobs/Yagudo_Persecutor.lua:14]: just did some digging and found some posts indicating all 3 yag NM are actually pure lotto, so my 1hr repop for all 3 is combo of luck and spawn rates I guess

Comment (TeoTwawki) [reply scripts/zones/Giddeus/mobs/Yagudo_Persecutor.lua:14]: ima leave well enough alone due to not knowing which sources I trust on this. you wanna edit that comment tho :wink:

Comment (CriticalXI) [reply scripts/zones/Giddeus/mobs/Yagudo_Persecutor.lua:14]: Would you consider the posts you found good enough evidence to give them all pure lotto cooldown?

Comment (TeoTwawki) [reply scripts/zones/Giddeus/mobs/Yagudo_Persecutor.lua:14]: I'm not sure how much I trust hoo mjuu true lotto. I think I could believe it on the other 2 even tho I never witnessed it any of my own camping sessions, but as camped as mjuu was I just find it hard to believe nobody ever caught it sooner. I used to love ruining the RMT's day there and even now on modern I swill sweep through all 3 NM there when I am bored (and the newer on they added near the BCNM end of the zone)

You can change Duzu if you want or leave as is - bg backs the true lotto claim on his wiki page

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**